.github/CODEOWNERS

```diff
@@ -55,6 +55,8 @@
 # PRLabel: %KeyVault
 /sdk/keyvault/                                      @g2vinay @vcolin7 @samvaity
 
+/sdk/loganalytics/microsoft-azure-loganalytics/     @divya-jay @geneh @alongafni
```

Comment:
Recommend you take the entirety of ```/sdk/loganalytics``` to keep across other changes that may impact your library.